### DISCUSSION TITLE
limit edge to vertex?

### DISCUSSION THREAD

**Discussion by jbockle** [Gremlinq/ExRam.Gremlinq#604] [Q&A] [answered]
I realize this may not be something specific to this library, but is there a way I can limit which edges a vertex can have? For instance if I have a `Person` vertex and a `Address` vertex, is there a gremlin native way or something available in this library to limit the `LivesAt` edge to only be created between these vertex labels?

**Comment by danielcweber** [comment, marked as answer]
Janusgraph might let you define a schema that'll support some notion of this. Otherwise, I'm not a aware of any general feature like this in any GraphDB.
